src/interfaces/node.h

```diff
@@ -9,6 +9,7 @@
 #include <amount.h>     // For CAmount
 #include <net.h>        // For CConnman::NumConnections
 #include <netaddress.h> // For Network
+#include <support/allocators/secure.h> // For SecureString
```

Comment:
Nit: found this code comment helpful; perhaps add it to the same `#include <support/allocators/secure.h>` line in src/dummywallet.cpp, src/interfaces/node.cpp, src/qt/askpassphrasedialog.h, and  src/qt/walletcontroller.h